processes/developmentSetup.py

```diff
@@ -4,6 +4,7 @@
 import os
 from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
 import requests
+import newrelic.agent
```

Comment:
I don't think we should instrument this process. It should only be run by people setting up test environments so it doesn't really tell us anything about how the app is running!